src/h3lib/lib/localij.c

```diff
@@ -319,14 +319,14 @@ H3Error localIjkToCell(H3Index origin, const CoordIJK *ijk, H3Index *out) {
     if (res == 0) {
         if (ijk->i > 1 || ijk->j > 1 || ijk->k > 1) {
             // out of range input
-            return E_FAILED;
+            return E_DOMAIN;
         }
 
         const Direction dir = _unitIjkToDigit(ijk);
         const int newBaseCell = _getBaseCellNeighbor(originBaseCell, dir);
         if (newBaseCell == INVALID_BASE_CELL) {
             // Moving in an invalid direction off a pentagon.
-            return E_FAILED;
+            return E_DOMAIN;  // i'm assuming this means the input values were invalid?
```

Comment:
No, I think this is `E_UNREACHABLE_CELL` again - a direction the algo can't handle.

Reply:
Would we just want an error denoting exactly what this comment is saying? `E_INVALID_PENTAGON_DIRECTION`? Or something like that?

Or is that exactly what you mean by `E_UNREACHABLE_CELL`?

Reply:
https://github.com/uber/h3/issues/627